url-shortener-db/url_shortener_db/base_62_convertion.py

```diff
@@ -0,0 +1,25 @@
+class Base62Convertion:
+    def __init__(self):
+        self.BASE62 = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+        self.BASE62_len = len(self.BASE62)
+
+    def encode(self, number):
+        if number == 0:
+            return self.BASE62[0]
+
+        arr = []
+        while number:
+            number, rem = divmod(number, self.BASE62_len)
+            arr.append(self.BASE62[rem])
+        arr.reverse()
+        return "".join(arr)
```

Comment:
Another option. Feel free to ignore
```suggestion
        return "".join(reversed(arr))
```